.github/workflows/docker-publish.yml

```
@@ -0,0 +1,61 @@
+    if: github.event_name == 'pull_request' && github.event.action == 'closed'
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+      contents: read
+
+    steps:
+      - name: Delete PR image
+        env:
+          GH_TOKEN: ${{ secrets.GHCR_PAT }}
+          PR_NUMBER: ${{ github.event.pull_request.number }}
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2046:warning:5:25: Quote this to prevent word splitting

(shellcheck)